config-model/src/main/java/com/yahoo/vespa/model/admin/Admin.java

```diff
@@ -194,7 +195,11 @@ public FileDistributionConfigProducer getFileDistributionConfigProducer() {
     public void addPerHostServices(List<HostResource> hosts, DeployState deployState) {
         if (slobroks.isEmpty()) // TODO: Move to caller
             slobroks.addAll(createDefaultSlobrokSetup(deployState.getDeployLogger()));
-        addMetricsProxyCluster(hosts, deployState);
+
+        // TODO: enable for all zones
+        if (deployState.zone().system() == SystemName.cd)
+            addMetricsProxyCluster(hosts, deployState);
+
```

Comment:
With a feature flag you could control this more fine-grained without risking braking anything

Reply:
Yes, but do we need it?

Reply:
You never know. I have never regretted adding one (since it's not that much work), but several times regretted not adding one or not being able to use one. Not having metrics would be an incident. In addition you can test this whenever it suits you and not worry about checking if this works when rolled out. But enabling this in CD only is fine with me